core/validatorapi/validatorapi.go

```diff
@@ -989,19 +989,22 @@ func (Component) NodeVersion(context.Context) (string, error) {
 	return fmt.Sprintf("obolnetwork/charon/%s-%s/%s-%s", version.Version, commitSHA, runtime.GOARCH, runtime.GOOS), nil
 }
 
-// convertValidators returns the validator map with all root public keys replaced by public shares.
-func (c Component) convertValidators(vals map[eth2p0.ValidatorIndex]*eth2v1.Validator) (map[eth2p0.ValidatorIndex]*eth2v1.Validator, error) {
+// convertValidators returns the validator map with root public keys replaced by public shares for all validators that are part of the cluster.
+func (c Component) convertValidators(vals map[eth2p0.ValidatorIndex]*eth2v1.Validator, allVals bool) (map[eth2p0.ValidatorIndex]*eth2v1.Validator, error) {
```

Comment:
this function should know about its use-cases, rather, variables should be names according to their meaning in the function. So rather `ignoreNotFound bool`